src/main/resources/templates/repo/appconfig-template.json.vpt

```diff
@@ -0,0 +1,41 @@
+
+    ,"${stack}${instance}AppConfigApp": {
+        "Type": "AWS::AppConfig::Application",
+        "Properties": {
+            "Name": "aws-appconfig-demo"
```

Comment:
The name, description and content can be parameters taken from a configuration (json) file that we can use as the "source of truth". We can even support multiple "appConfig" instances if in the future we decided to use this more broadly.